builtin/logical/plugin/backend.go

```diff
@@ -0,0 +1,39 @@
+package plugin
```

Comment:
What would be the correct action to take in this case? How it would enable restarting the plugin process.